assets/scss/base/form/_form.scss

```diff
@@ -227,6 +227,18 @@ Styleguide Form.date
   @include ie-lte(7) {
     zoom: 1;
   }
+
+  @include media(mobile) {
+    .form-group {
+      width: 100%;
+    }
+  }
+
+  .form-field--inline-text-input {
```

Comment:
This isn't needed if `form-field--inline-text-input` is removed in https://github.com/hmrc/play-ui/pull/44